test/unit/presenters/publishing_api/world_index_presenter_test.rb

```diff
@@ -2,9 +2,30 @@
 
 class PublishingApi::WorldIndexPresenterTest < ActiveSupport::TestCase
   setup do
-    @world_location_1 = create(:world_location, name: "Location 1", slug: "location-1", active: true)
-    @world_location_2 = create(:world_location, name: "Location 2", slug: "location-2", active: false)
-    @international_delegation = create(:international_delegation, name: "Delegation", slug: "delegation", active: true)
+    @world_location_1 = create(
+      :world_location,
+      name: "Location 1",
+      slug: "location-1",
+      active: true,
+      analytics_identifier: "A1",
+      iso2: "AB",
+    )
+    @world_location_2 = create(
+      :world_location,
+      name: "Location 2",
+      slug: "location-2",
+      active: false,
+      analytics_identifier: "B2",
+      iso2: "CD",
+    )
+    @international_delegation = create(
+      :international_delegation,
+      name: "Delegation",
+      slug: "delegation",
+      active: true,
+      analytics_identifier: "C3",
+      iso2: nil,
```

Comment:
I've left this value present, but `nil` rather than just not including the key, as this reflects the existing behaviour of the Whitehall API.